jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -335,6 +342,35 @@ def _roman_2d_to_glue_data(file_obj, data_label, ext=None):
         yield data, new_data_label
 
 
+def _asdf_2d_to_glue_data(file_obj, data_label, ext=None):
```

Comment:
ASDF is too generic and this is specifically about Roman, so the function name should be more specific. For example, ASDF from DKIST is going to crash here.

```suggestion
def _roman_asdf_2d_to_glue_data(file_obj, data_label, ext=None):
```